templates/compose/nodebb.yaml

```diff
@@ -0,0 +1,60 @@
+# ignore: false
+# documentation: https://docs.nodebb.org/
+# slogan: A next-generation discussion platform.
+# tags: communication, forums, discussion
+# logo: svgs/nodebb.svg
+# port: 4567
+
+services:
+  nodebb:
+    image: ghcr.io/nodebb/nodebb:latest
+    restart: unless-stopped
+    environment:
+      - SERVICE_FQDN_nodebb_4567
```

Comment:
Please use all caps for environment variables

Reply:
Dumb mistake, fixed it.